docs/frameworks/nitrojs.md

```diff
@@ -0,0 +1,246 @@
+---
+description: Learn how to deploy a Nitro.js application with Genezio.
+---
+
+import Tabs from '@theme/Tabs';
+
+import TabItem from '@theme/TabItem';
+
+# Nitro.js
+
+<head>
+    <title>Nitro.js | Genezio Documentation</title>
+</head>
+
+Nitro.js is a modern framework for building server-side applications, designed to optimize performance and streamline development.
+It offers a comprehensive set of features for creating web servers and APIs, supporting multiple deployment targets. 
+
+# Deployment
+
+In this tutorial, you will learn how to deploy an existing Nitro.js app using Genezio, a serverless deployment platform that simplifies app management and reduces costs. We'll cover the benefits, answer common questions, and provide detailed deployment steps.
```

Comment:
Looks like this file is more of a tutorial, instead of a framework documentation. In my opinion a framework documentation should not have this personal touch and should be more technical.

Reply:
This is taken from the express.js framework documentation file, this is why I put it in, but we can take it off.

Reply:
You should consult a second opinion on this, maybe I'm biased. Ultimately, it's your call if you keep it this way, change it or remove it.

Reply:
Talked to Paula and I agree that we should delete that part. In the following days we should make the same changes to express, and agree to one common template for the frameworks, since the documentation for next and express are written very differently, and it looks inconsistent.